services/workflows-service/package.json

```diff
@@ -138,6 +139,7 @@
     "jest": "29.7.0",
     "jest-mock-extended": "^2.0.4",
     "json-schema-to-typescript": "^13.0.1",
+    "libpg-query": "^15.0.2",
```

Comment:
are you sure you meant to install it as **dev** dependency?

Reply:
yes